EasyIntern-Backend/Program.cs

```diff
@@ -76,7 +76,7 @@
 app.UseRouting();
 app.UseHsts();
 app.UseSession();
-app.UseHttpsRedirection();
+// app.UseHttpsRedirection();
```

Comment:
Why is this commented?
Thought this was necessary to use the Redirect and RedirectToAction methods in controllers

Reply:
I'm not sure how it affects Redirect and RedirectToAction however, since I am not familiar with asp.net

Reply:
Tested it, it worked fine without the UseHttpsRedirection method